app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Student finance for teacher training"
-heading: "Tuition fee and maintenance loans"
+heading: "Student finance for teacher training"
 description: |-
-    Find out about the available student finance for teacher training, including tuition fee and maintenance loans to support you while you train.
+    Find out about the available student finance for teacher training, including tuition fee loans and maintenance loans to support you while you train.
 related_content:
```

Comment:
We link to the Find promo (line 10) at the end of this page which seems slightly odd - better for people to get to the adviser promo?

Reply:
I'll delete it as we've got the adviser under it.